extensions/rocksdb-repos/FlowFileRepository.cpp

```diff
@@ -89,23 +82,28 @@ void FlowFileRepository::flush() {
   }
 }
 
-void FlowFileRepository::run() {
-  // threshold for purge
+void FlowFileRepository::printStats() {
```

Comment:
If I understand correctly, this is ran every purge_period, which is, by default, 2.5 s.
4 log lines every 2.5 seconds is way too much noise, affecting everyone, as virtually everyone uses this FlowFileRepo and the default log level is info. This would account for the majority of logs in many cases.
I think this should definitely be at debug level.

Reply:
Fair point.

I would keep it at info level, but call less frequently.

Reply:
I'd also suggest logging the stats in a single line. It reduces log spam while keeping usefulness.

Reply:
Now it's single line.